index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Comforter+Brush&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+<h1 id="title" class="text-center">Survey Form</h1>
+<p id="description">Thank you for taking the time to fill out this form </p>
+
+
+<form action="#" id="survey-form">
+
+
+<label for="name" id="name-label">Enter your name:</label>
+<input type="text" id="name" placeholder="Enter a Name" required> <br> <br>
+
+
+
+<label for="email" id="email-label">Enter your email address:</label>
+<input type="email" id="email" placeholder="Enter an e-mail" required > <br><br>
+
+<label for="phone-number" id="number-label">Enter your phone-number:</label>
+<input type="number" id="number" placeholder="+1 222 222 2222" min="10" max="99" required><br> <br>
```

Comment:
Please refrain from using line breaks. `<br>`.
Instead, use the styling sheet to create spaces.